force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.js

```diff
@@ -328,22 +328,6 @@ export default class RecurringDonationTable extends LightningElement {
                     return { actions, ...el, nexDonationFormatFirstElement, nexDonationFormatSecondElement, lastModifiedDate };
                 });
             }
-        }).finally(() => {
-          this.data?.forEach((item) => {
-            let nextDonationHtml = `<div class="${this.rowClasses}">`;
-            if(item.recurringDonation.npe03__Next_Payment_Date__c !== "Invalid Date"){
-                if(item.nextDonation){
-                    item.nextDonation.split(',').forEach((nextDonationElement) => {
-                      nextDonationHtml += `${nextDonationElement} </br>`
```

Comment:
Adding the breakpoint is the only logic I could find in this, so I reimplemented that via Apex.